integration-tests/utils/integration-helpers.js

```diff
@@ -74,34 +72,20 @@ const withUser = test => async args => {
   }
 }
 
-const addUserToBilling = withUserToken(async ({ billingProject, context, email, testUrl, token }) => {
-  const ajaxPage = await context.newPage()
-
-  await ajaxPage.goto(testUrl)
-  await signIntoTerra(ajaxPage, token)
-
-  await ajaxPage.evaluate((email, billingProject) => {
+const addUserToBilling = _.flow(withSignedInPage, withUserToken)(async ({ page, billingProject, email }) => {
```

Comment:
TOL question: Any reason why you chose to keep this `flow` on one line?

Reply:
No particular reason except that it seemed short and simple enough to be readable that way.